ports/hawkisland/Hawk Island.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+GAMEDIR="/$directory/ports/hawkisland"
+
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
+
+cd $GAMEDIR
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+$GPTOKEYB "gmloadernext" &
+pm_platform_helper "$GAMEDIR/gmloadernext"
+
+$ESUDO chmod +x "$GAMEDIR/gmloadernext"
+
+./gmloadernext game.apk
```

Comment:
gmloadernext doesn't use this command line argument to load the apk, it's always game.apk. (see [here](https://github.com/JohnnyonFlame/gmloader-next/blob/6f2569914109866c1fb903a9b7aed970896b5618/gmloader/main.cpp#L83))
```suggestion
./gmloadernext
```